### ISSUE TITLE
History loading each time after opening Postman 8.xx 🐞

### ISSUE THREAD

**Issue by vivekpatel111** [a85/POSTMan-Chrome-Extension#9566]
**Describe the Issue**

- I've recently upgraded to Postman 8.x version, which has brought way many features then previous versions. However one downside to this update was like, history tab was always fetching requests when I open Postman.
- Due to this issue, I was not able to find my previous requests (even consider request from last 2 days - not even considering last 2 weeks request) which was search-able in previous versions.

[Edited]
- Found out one more issue - search term is not being retained while clicking on history and collections and vice versa (earlier it was retained)

**Steps to Reproduce**

- None

**Screenshots or Videos**

- None

**Environment Information**

 - Platform Type: [e.g. Native Desktop App, Postman On The Web, Web Dashboard] Postman for Mac
 - Postman Version: Version 8.0.4
 - Operating System: OS X 20.3.0 / x64

**Additional Context**

- None

**Comment by vivekpatel111** [comment]
As a developer, I feel Postman is great product which is helping quite a lot in day to day activities. As we're using it on daily basis, it's very useful for us to go through on history for digging older requests.

Here's possible solutions (which I think can be used) :

1. Use scratchpad instead of web version (this means no data will be synced across devices)
2. Store history data locally for web version
3. Restore previous history implementation
4. Downgrade Postman to 7.x or later versions (horrible but quickest solution)

Let me know what you can think on these. 😃 #peace

**Comment by vivekpatel111** [comment]
Final solution : Downgrade Postman to 7.x or later versions 😢 

Click [here](https://dl.pstmn.io/download/version/7.36.4/osx64) to download latest 7.x version.

**Comment by tim-hanssen** [comment]
The most annoying thing while developing!

**Comment by KaushikJain** [comment]
I downgraded to 7.x version as directed by support and now my local history is deleted as well. Web portal shows the history but there is no direct way to export the same or get it in my local app.

**Comment by jeffbendel** [comment]
This is a giant time suck. If I try searching for certain posts, it won't even pull the history. I have to wait for every request to load across months, and then search for the one post I need

**Comment by KaushikJain** [comment]
Any ETA on this? It is frustrating to wait for the history to load and sometimes waiting doesn't help. Then have to restart postman and then wait again. Postman web is better for searching than the App most of the time. 

https://user-images.githubusercontent.com/22023419/115814421-9545ee00-a427-11eb-84b9-6fad4871b4ad.mov

**Comment by vivekpatel111** [comment]
> Any ETA on this? It is frustrating to wait for the history to load and sometimes waiting doesn't help. Then have to restart postman and then wait again. Postman web is better for searching than the App most of the time.
> 
>  Screen.Recording.2021-04-23.at.11.30.22.AM.mov

@KaushikJain @jeffbendel 

I can understand your concerns, but if you think from their side then this new release is major version release for them and it's expected to have some issues/unknowns coming up in such things. I'm sure they are trying hard to resolve most of the issues asap. In the meantime, you should downgrade if you're depending upon it (like me) and not wait for them to fix it in near times. #peace 🤝

**Comment by kirillrocks** [comment]
@vivekpatel111 Hey, we are suffering from the issue as well, it is impossible to use the history as it is. Can you share the link to version 7 for Windows?

Thanks in advance!

**Comment by vivekpatel111** [comment]
@kirillrocks You can download latest 7.x windows version from [here](https://dl.pstmn.io/download/version/7.36.4/win64)

**Comment by joelrozen** [comment]
Any word on a fix for this yet? It only started for me after updating Postman, and some days I make so many requests that I can't even search for something I made yesterday!!!! It takes me 10+ minutes of scrolling to be able to search for requests I made a month ago, and I tried downgrading back to 7.x but that would only load the last 48 hours of requests and nothing else at all. 

Please there has to be a patch coming out soon this issue is almost 6 months old.

**Comment by jordanhenderson** [comment]
This is a very, very annoying bug. Please fix.

**Comment by DavydeVries** [comment]
@giridharvc7 / @vvs11 do we have an update on this?

**Comment by nacho85** [comment]
Somene from Postman please solve this. Is terribly annoying for developers... downgrading version to 7.x.
I don't know why they do such things... or at least give me the option to get the history on demand (or not), but don't apply that by default... Hope this get solved soon

**Comment by vivekpatel111** [comment]
I don't know why is it not prioritised or what's blocker for this one. This is something which must be picked up asap as this is one of the core functionality of the product. Any thoughts @giridharvc7 @vvs11 

#disappointed #bugs #critical #blocker #core

**Comment by ghabre** [comment]
I also reported the same issue at some point and ended up downgrading to version 7

I even figured out a way in version 7 to copy my history across PCs, this quick all history search is one my most used features in postman

The latest postman 7 can be downloaded from: [https://dl.pstmn.io/download/version/7.36.5/win64](https://dl.pstmn.io/download/version/7.36.5/win64)

**Comment by saezf** [comment]
Really annoying bug. Is there a fix planned for the next release?
I get that the workaround is to downgrade to 7x, but it would be nice if we can get an actual fix for this bug.

**Comment by AGVeem** [comment]
It is 2022 over here...  seriously, it's been a year we are missing one of the basics functionality

**Comment by giridharvc7** [comment]
@vivekpatel111 and others, sorry for not communicating this earlier, this has been fixed in 9.14+ versions (we are in v10 right now) can you please confirm if you still face this issue?

**Comment by jgillard** [comment]
Yes I definitely still face this issue on every app load; MacOS v10.2.0

**Comment by TinTran64** [comment]
It's been almost 2 years now and this issue is still open...

**Comment by camoore** [comment]
Still have this issue. Possibly due to the infinite scroll implementation. Scrolling up and back down loads more of the history

#11497 

#9513

**Comment by giridharvc7** [comment]
Can you share some more details on this? A screen recording will be really helpful - also can you let us know if you are on scratchpad or signed into a workspace?

**Comment by TinTran64** [comment]
@giridharvc7 I'm on the latest postman 10.8.1 and if I don't collapse the history or scroll down for the history to load I can't search my request if the request I'm looking for hasn't loaded yet 

https://user-images.githubusercontent.com/50310568/213070241-3896535c-df6a-4d67-a961-ad84cb4d5d2f.mov

**Comment by vivek-osmos** [comment]
@giridharvc7 is there any workaround for this in any newer versions or is it already solved?
